v4/source/posts.yaml

```diff
@@ -397,6 +397,49 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/users/{user_id}/channels/{channel_id}/posts/unread':
+    get:
+      tags:
+        - posts
+      summary: Get posts for a channel around last unread
+      description: |
+        Get posts in a channel around last unread post of a member.
```

Comment:
Can we try to make this description more descriptive? To me it's not exactly clear what the purpose of this endpoint is

Reply:
Not sure if this gets any better.  I'm thinking something like:
```yaml
summary: Get posts for a channel around last unread
description: |
  This endpoint determines the last unread post of a member to the channel and gets a number of posts before and after the said post. Use the query parameters `limit_before` and `limit_after` to modify the number of posts to be returned by this endpoint. 
  __Minimum server version: 5.3__
  ##### Permissions
  Must have `read_channel` permission for the channel.
```

Let me know if that sounds good.  Would appreciate suggestion into it.  Thanks

Reply:
`the said post` should just be `said post` or even `that post`. Also, I think you can leave out the explanation of the query parameters since they're explained below